packages/components/Menu/src/MenuCallout/MenuCallout.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
This file is a move without edits - or there are changed here?

Reply:
This is entirely newly created folder called as MenuCallout based on feedbacks from Ruriko and Amanda. 
Thats might be the reason not seeing any edits.